src/components/Login.js

```diff
@@ -0,0 +1,90 @@
+import React, {Component} from 'react';
+import User from './User.json';
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      password: '',
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+
+  handleInputChange = event => {
+    // will update state by name attribute, so state name should be same with it
+    const {value, name} = event.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  onSubmit = event => {
+    event.preventDefault();
+
+    if (
+      User.find(data => {
+        return data.email === this.state.email; // if can't find, it will return undefined
+      }) != undefined &&
+      User.filter(data => {
+        return data.password === this.state.password;
+      }) != undefined
+    ) {
+      alert('logged in!!');
```

Comment:
Can you change this alert to "Login successful"?

Reply:
thanks for review! I changed the message